apps/qubit/modules/informationobject/actions/browseAction.class.php

```diff
@@ -503,8 +503,10 @@ protected function setFilterTags($request)
         // Set label for has digital object filter tag
         if (filter_var($request->onlyMedia, FILTER_VALIDATE_BOOLEAN)) {
             $this->setFilterTagLabel('onlyMedia', $i18n->__('With digital objects'));
-        } else {
+        } elseif (!filter_var($request->onlyMedia, FILTER_VALIDATE_BOOLEAN) && '0' == $request->onlyMedia) {
```

Comment:
Again, `!filter_var($request->onlyMedia, FILTER_VALIDATE_BOOLEAN)` is redundant here.

Reply:
And the strict comparison operator should be used for `'0' === $request->onlyMedia`